test/integration/targets/vmware_host_hyperthreading/tasks/main.yml

```diff
@@ -0,0 +1,83 @@
+# Test code for the vmware_host_acceptance module.
+# Copyright: (c) 2018, Abhijeet Kasurde <akasurde@redhat.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+# TODO: vcsim does not support Acceptance Level related to operations
```

Comment:
s/Acceptance Level/Hyperthreading/

Reply:
Sorry, I copied the tests from one of your modules.